packages/calcite-components/src/components/navigation-logo/navigation-logo.tsx

```diff
@@ -107,34 +113,45 @@ export class CalciteNavigationLogo implements LoadableComponent {
     return <calcite-icon class={CSS.icon} flipRtl={this.iconFlipRtl} icon={this.icon} scale="l" />;
   }
 
+  renderHeaderContent(): VNode {
+    const { heading, headingLevel, description } = this;
+    const headingNode = heading ? (
+      <Heading class={CSS.heading} level={headingLevel}>
+        <span
+          aria-label={this.heading}
+          class={{
+            [CSS.heading]: true,
+            [CSS.standalone]: !this.description,
+          }}
+          key={CSS.heading}
+        >
+          {heading}
+        </span>
+      </Heading>
+    ) : null;
+
+    const descriptionNode = description ? (
+      <span aria-label={this.description} class={CSS.description} key={CSS.description}>
```

Comment:
Whats the need for the aria-label here as well?

Reply:
I was just trying to keep the same sematics as what was in the original code, but I've removed it.